src/software/run_simulated_ai_vs_ai.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# The name of this script
+THIS_SCRIPT_FILENAME=$(basename "$0")
+
+# The number of arguments passed to this script
+NUM_ARGS=$#
+
+if [ $NUM_ARGS -ne 1 ]; then
+  echo "Error: Incorrect number of arguments to script"
+  echo "Usage: $THIS_SCRIPT_FILENAME [interface_name]"
+  echo "Choose from the following interfaces:"
+  echo "`ls /sys/class/net`"
+  exit 1
+fi
+
+if [[ "$1" == "help" ]]; then
+  echo "Runs AI vs AI with the standalone simulator"
+  echo "Usage: $THIS_SCRIPT_FILENAME [interface_name]"
+  echo "Choose from the following interfaces:"
+  echo "`ls /sys/class/net`"
+  exit 0
+fi
+
+bazel run //software:full_system -- --backend=WifiBackend --interface=$1 &
+bazel run //software:full_system -- --backend=WifiBackend --interface=$1 &
+bazel run //software/simulation:standalone_simulator_main -- --interface=$1 &
```

Comment:
Here is the promised tmux method, you should be able to click on the windows

~~~bash
tmux new-session \; \
  set -g mouse on \; \
  send-keys "bazel run //software:full_system -- --backend=WifiBackend --interface=$1" C-m \; \
  split-window -h \; \
  send-keys "bazel run //software:full_system -- --backend=WifiBackend --interface=$1" C-m \; \
  split-window -v \; \
  send-keys "/tmp/ssl-game-controller_v2.4.0_linux_amd64" C-m \; \
  select-pane -t 0 \; \
  split-window -v \; \
  send-keys "bazel run //software/simulation:standalone_simulator_main -- --interface=$1" C-m \;

~~~